guides/README.md

```diff
@@ -6,7 +6,7 @@ Read about the basics of Safe and how it compares to other solutions [here](http
 
 ## Safe Core SDK
```

Comment:
Not sure if this guide make sense instead using the docs.

In any case I will do one of the following with the `integrating-the-safe-core-sdk.md` guide and corresponding

1) Update the guide for reflecting our usual naming:
- safeService -> apiKit
- safedk -> protocolKit
- Safe.create -> Safe.init
2) Remove README and guide as the docs are probably covering the same info

Reply:
I went to option 1 for now. I would still leave the guide around as it's updated to the latest changes.

We don't have such guide in the docs yet so I would leave it here as an example that we can port later.